src/js/slider.js

```diff
@@ -0,0 +1,89 @@
+document.addEventListener('DOMContentLoaded', () => {
+  let activeButton = true
+  let slideIndex = 0
+  const slider = document.getElementById('slides')
+  const arrowLeft = document.getElementById('arrowLeft')
+  const arrowRight = document.getElementById('arrowRight')
+  const slides = [...slider.children]
+  const slidesCount = slides.length
+
+  const setSliderHeight = () => {
+    const height = getComputedStyle(slides[0]).height
+    slider.style.height = height
+  }
+
+  setSliderHeight()
+  window.addEventListener('resize', setSliderHeight)
+
+  slides.forEach((slide, index) => {
+    if (index != slideIndex) {
+      slide.classList.add('hidden')
+    }
+  })
+
+  arrowLeft.addEventListener('click', () => {
+    if (!activeButton) return
+    activeButton = false
+    const nextSlideIndex = slideIndex === 0 ? slidesCount - 1 : slideIndex - 1
+
+    const s1 = slides[slideIndex]
+    const s2 = slides[nextSlideIndex]
+
+    const width = slider.clientWidth
+
+    s2.style.left = `${width}px`
+    s1.style.left = '0px'
+    s2.classList.remove('hidden')
+
+    const step = width / 200
```

Comment:
Replace 200 to a constant with appropriate name. It is difficult to understand what 200 means.